Makefile.am

```diff
@@ -52,6 +52,7 @@ myhtopsources = \
 	Panel.c \
 	Process.c \
 	ProcessList.c \
+	ProcessLocksScreen.c \
```

Comment:
Just occurred to me this functionality is Linux-specific, so the source should live below the linux/ subdirectory.